### DISCUSSION TITLE
Missing "Update Core" under Global Settings

### DISCUSSION THREAD

**Discussion by mfore1** [MasaCMS/MasaCMS#278] [Q&A]
Hi folks!
Somehow the update feature is not available in my MASA installation. Is there any way to install or activate it? 
Thanks!

**Comment by guustnieuwenhuis** [comment]
What version of Mura/Masa CMS ar you on?

**Comment by mfore1** [comment]
I updated to the latest version of Masa manually, so it is 7.4.5. 
This is an older installation I migrated from Mura CMS. 
May there be some setting I need to ajust so the updater works fine?
